sys/usb/usbus/cdc/acm/cdc_acm.c

```diff
@@ -273,13 +292,15 @@ static void _handle_in(usbus_cdcacm_device_t *cdcacm,
         (cdcacm->state != USBUS_CDC_ACM_LINE_STATE_DTE)) {
         return;
     }
+    irq_disable();
```

Comment:
Tiny nitpick: Maybe add a comment here to indicate that the loop below is bounded at `USBUS_CDC_ACM_BULK_EP_SIZE` iterations.

Reply:
Done.